composer.json

```diff
@@ -28,7 +28,8 @@
     },
     "require-dev": {
         "producer/producer": "^2.3",
-        "phpunit/phpunit": "^9.5",
+        "yoast/phpunit-polyfills": "^1.0",
+        "phpunit/phpunit": ">=8.0",
```

Comment:
This dependency is way too broad in my opinion: `"phpunit/phpunit": "^8.0 || ^9.0"` would be my suggestion. Make explicit what we support.

Reply:
If you want I can make it, but this `phpunit/phpunit` was only added because of a failing test for lowset build . See https://github.com/auraphp/Aura.Di/pull/206/checks?sha=3b338fee88782f3b633a7cd32ec813fef4b9f23a#step:5:17  it is installing 4.x version of php. 

Why I am so confident about the phpunit is we are using `yoast/phpunit-polyfills` .  So internally it is depending on that class. So it will make sure there is no issues  or breaks happen even if phpunit breaks at 10.x .

Reply:
@frederikbosch I have updated phpunit dependency.